core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1398,7 +1398,7 @@ account.menu.notifications=Notifications
 
 account.menu.walletInfo.balance.headLine=Wallet balances
 account.menu.walletInfo.balance.info=This shows the internal wallet balance including unconfirmed transactions.\n\
-  For Bitcoin the sum of the 'available balance' and the 'reserved for offers balance' must match the internal wallet balance \
+  For Bitcoin the sum of the 'available balance' and the 'reserved in offers balance' matches the internal wallet balance \
   displayed here.
```

Comment:
If the suggestion in https://github.com/bisq-network/bisq/pull/4980/files#r546901592 is accepted, then this line should be removed.

```suggestion

```